src/AuthenticationResponse.ts

```diff
@@ -39,6 +41,16 @@ export default class AuthenticationResponse {
     verifiedJwt: SIOP.VerifiedAuthenticationRequestWithJWT,
     responseOpts: SIOP.AuthenticationResponseOpts
   ): Promise<SIOP.AuthenticationResponseWithJWT> {
+    const pejs: PEJS = new PEJS();
+    // bunch of ifs to make sure we need to call evaluate
+    const pd: PresentationDefinition =
+      verifiedJwt.payload.claims['id_token']['verifiable_presentations'].presentation_definition;
```

Comment:
Needs to be a bit more defensive. A definition doesn't have to be there, let alone the claims values/keys